src/bin/startGroovy

```diff
@@ -261,18 +261,18 @@ startGroovy ( ) {
     if $useprofiler ; then
         runProfiler
     else
-        exec "$JAVACMD" $JAVA_OPTS \
-            -classpath "$STARTER_CLASSPATH" \
-            -Dscript.name="$SCRIPT_PATH" \
-            -Dprogram.name="$PROGNAME" \
-            -Dgroovy.starter.conf="$GROOVY_CONF" \
-            -Dgroovy.home="$GROOVY_HOME" \
-            -Dtools.jar="$TOOLS_JAR" \
+        eval exec "\"\$JAVACMD\"" $JAVA_OPTS \
+            -classpath "\"\$STARTER_CLASSPATH\"" \
+            -Dscript.name="\"\$SCRIPT_PATH\"" \
+            -Dprogram.name="\"\$PROGNAME\"" \
+            -Dgroovy.starter.conf="\"\$GROOVY_CONF\"" \
+            -Dgroovy.home="\"\$GROOVY_HOME\"" \
+            -Dtools.jar="\"\$TOOLS_JAR\"" \
```

Comment:
Looks like there is an extra backslash before the environment variable that will cause it to not interpolate.  For example, think change should just be:

`"\"$VAR\""` (no slash before `$`)

For reference I checked [Tomcat's catalina.sh] (https://github.com/apache/tomcat/blob/86b1281a190ec2f17fd1ac5ae3d3889166dcd360/bin/catalina.sh#L325-L339) to see how they handled it.